README.md

```diff
@@ -37,13 +37,13 @@ Install-Module -name ADSIPS
 ### Download from GitHub repository 
 
 1. Download the repository
-1. Unblock the zip file
-1. Extract the folder to a module path (e.g. $home\Documents\WindowsPowerShell\Modules)
-1. Run `build.ps1` (exists in project root)
+2. Unblock the zip file
```

Comment:
Incremented numbers are not needed in Markdown.
Markdown will automatically increment when generating the HTML.
This allows you to easily move things around.

I'll fix that in next commit.

Reply:
While that is correct - the 'bullet point' causes the count to reset.. This is what I was resolving:

<img width="841" alt="Screen Shot 2019-06-23 at 12 55 26 PM" src="https://user-images.githubusercontent.com/21092343/59980152-67878800-95b7-11e9-888b-0d73f1753534.png">

Reply:
fixed, was missing a space